### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by rudimusmaximus**
basic test edit

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/rudimusmaximus/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

**Comment by rudimusmaximus** [comment]
Looks good

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome  
+
```
Comment (rudimusmaximus) [inline ReadMe.md]: be more specific, but good to merge. !completed  !completed

Comment (rudimusmaximus) [inline ReadMe.md]: !fix

**Pull request merged**